app/modules/onboardin/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.js.map
```

Comment:
For what? Do you use `npm start`?

Reply:
I do, but as I am new in TypeStript I compile .ts to .js to understand how it works under the hood.

Reply:
Ah, ok, the [Playground](https://www.typescriptlang.org/play/) is also good for it.